### PR TITLE
Change workspace hostname config scheme

### PR DESCRIPTION
**Pull request by tdy**

### CODE REVIEW

File: workspace_host/interface.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const ERR = require('async-stacktrace');
 const express = require('express');
 const app = express();
 const request = require('request');
-const port = 8081;
+const PORT = 8081;
```
Comment (tdy) [inline workspace_host/interface.js:5]: Is this supposed to be config'ed?

File: workspace_host/interface.js [hunk 2]
```diff
@@ -149,7 +148,7 @@ app.post('/', function(req, res) {
     }
 });
 
-app.listen(port, () => console.log(`Listening on http://${config.workspaceNativeLocalhost}:${port}/`));
+app.listen(PORT, () => console.log(`Listening on http://${config.workspaceDevContainer}:${PORT}/`));
```
Comment (tdy) [inline workspace_host/interface.js:151]: `PORT` used here (should we be config'ing this?)

File: lib/config.js [hunk 3]
```diff
@@ -110,9 +110,8 @@ config.hasLti = false;
 config.ltiRedirectUrl = null;
 config.filesRoot = '/files';
 config.workspaceS3Bucket = null;
-config.workspaceContainerLocalhost = 'localhost';
-config.workspaceContainerPort = 8081;
-config.workspaceNativeLocalhost = 'host.docker.internal';
+config.workspaceDevHost = 'localhost:8081';
```
Comment (mwest1066) [inline lib/config.js:113]:
```suggestion
config.workspaceDevHostName = 'localhost';
config.workspaceDevHostPort = '8081;
```

File: workspace_host/interface.js [hunk 4]
```diff
@@ -2,7 +2,8 @@ const ERR = require('async-stacktrace');
 const express = require('express');
 const app = express();
 const request = require('request');
-const port = 8081;
+// TODO: change to be configurable
+const PORT = 8081;
```
Comment (mwest1066) [inline workspace_host/interface.js:6]: Use `config.workspaceDevHostPort`

**Pull request merged**